docs/releases/5.2.txt

```diff
@@ -285,6 +285,12 @@ backends.
 
 * ...
 
+* The new :meth:`~django.db.models.Model._is_pk_set()` method is added to
+  support checking whether the ``pk`` property (composite or not) has been set:
+
+  * ``obj.pk is not None`` -> ``obj._is_pk_set()``
+  * ``obj.pk is None`` -> ``not obj._is_pk_set()``
```

Comment:
Given that this is still an "experimental" method, I would make this note more succinct:
```suggestion
* The new :meth:`~django.db.models.Model._is_pk_set()` method allows to
  check whether a Model instance has its primary key set or not.
```